build.gradle

```diff
@@ -236,16 +247,23 @@ static void assertExecutableSpecRequirementsForTests(Project project, Test test)
 
 project(":functionaltests") {
     dependencies {
-        compile project(":service")
+        testImplementation project(":service")
+
+        testImplementation "org.concordion:concordion:${depVersions.concordion}"
```

Comment:
they separate now dependencies which should be exposed and dependencies which should be just included, but not available for a library consumers.
https://docs.gradle.org/current/userguide/java_library_plugin.html
https://stackoverflow.com/a/44493379/907576

in short: 
  - if we make a library, we should make _api_ those dependencies, which should be transitive and available for our library consumers. The dependencies which should be available only for use should bi _implementation_
  - if we make a web-services (as this project is) - everything should b _implementation_ since we don't expose Java dependencies to consumers at all